server/analytics/event.go

```diff
@@ -260,7 +261,7 @@ func (a Analytics) GetLastDaysEventCount(organizationID uuid.UUID) (int, error)
 }
 
 const deleteEvent = `
-ALTER TABLE events_local ON CLUSTER attractify
+ALTER TABLE events%s %s
```

Comment:
I would create an helper method that builds my table statement.

```
func (a Analytics) tableName(name string) string {
        if a.ClusterArgs == nil {
                return name
        }
        return fmt.Sprintf("%s_%s ON CLUSTER %s", name, a.CulsterArgs.LocalSuffix, a.ClusterArgs.ClusterName)
}
```

And then write

```
ALTER TABLE ?
```

and interpolate ? with a.tableName("events") if you could interpolate table names. Otherwise use `%` and fmt.Sprintf(a.tableName("events"))`.